samples/msal-node-samples/ElectronSystemBrowserTestApp/README.md

```diff
@@ -68,8 +69,8 @@ cd samples/msal-node-samples/standalone-samples/ElectronSystemBrowserTestApp
    1. Replace the string `Enter_the_Cloud_Instance_Id_Here` with `https://login.microsoftonline.com`.
    1. Replace the string `Enter_the_Graph_Endpoint_Here`. with `https://graph.microsoft.com/v1.0`.
 
-1. Open the [.authConfig.json](./redirect/public/config/authConfig.json) file and provide the required configuration value
-   1. Replace the string `Enter_the_default_Redirect_Uri_Here`  with the default redirect uri generated by portal `msal{Your_Application/Client_Id}://auth`.
+1. To enable deep linking in the application, open the [successTemplate.html](./public/successTemplate.html) file and provide the required configuration value:
+   1. Fill in your application/client ID in the string `msal{Your_Application/Client_Id}://auth`. If you like, you can replace this value with any unique string.
```

Comment:
A bit confused by the last sentence here. Can it really be _any_ string? If so, what constitutes as "unique"?

Reply:
I updated the description and added a link to electron docs.